### DISCUSSION TITLE
Add custom 'Worknote Snippets' to the context menu

### DISCUSSION THREAD

**Discussion by alonsotorres** [arnoudkooi/SN-Utils#384] [Ideas]
I really like the '**Worknote Snippets**' option from the context menu for quickly adding comments to journals.

However, I'm wondering if there are any plans on making them customizable by users.

It would be really helpful to have a custom pre-defined list of snippets with more than just the '**Code**' and '**Hypelink**' options.

![Worknote smippets](https://user-images.githubusercontent.com/643244/230810281-ec0def31-ec14-4dd1-9907-d2e570a4d21d.png)

**Comment by vedang444** [comment]
I've saved a custom version with the code and hyperlink to my template and populate it directly from there. Something to try maybe?

**Comment by Anduh** [comment]
With the release of v10 in April 2026, [Code Snippets](https://snutils.com/docs/guide/snippets/snippets) where introduced as a Pro feature.

By double-pressing Shift and having any field or journal field in focus, the Snippets-menu shows up, where you can scroll or type to find your custom snippets. There is also a snippet marketplace that contains packs for a number of snippets you can add to your snippets so you don't even have to create all your snippets yourself.

You can also create new snippets with `/newsnip` command.

@vedang444
